drainpipe-dev/composer.json

```diff
@@ -48,7 +48,7 @@
                 "[web-root]/sites/sites.php": "scaffold/nightwatch/sites.php",
                 "[project-root]/.ddev/docker-compose.selenium.yaml": "scaffold/nightwatch/docker-compose.selenium.yaml",
                 "[project-root]/test/nightwatch/example.nightwatch.js": "scaffold/nightwatch/example.nightwatch.js",
-                "[project-root]/phpcs.xml": "scaffold/phpcs.xml"
+                "[project-root]/phpcs.xml.dist": "scaffold/phpcs.xml.dist"
```

Comment:
Have we renamed an existing scaffolding file before? We have encountered errors with this on previous projects when the root project ends up having any kind of dependency or sub-dependency on `composer/composer`: https://www.drupal.org/project/drupal/issues/3266399

I'd like to make sure this is tested with a manual composer update to confirm it's okay to rename. It might be *safer* to leave the existing file in the source repository here and essentially have duplicate files, but only one listed in the scaffolding itself.

Reply:
Took me a bit of time to figure out how to test with the existing test framework. I have tested out what happens when the upgrade happens. I have tested upgrading and all it seems to do is create a phpcs.xml.dist file. Not seeing any errors.